roles/katello_repositories/tasks/main.yml

```diff
@@ -8,6 +8,5 @@
   args:
     warn: false # dnf module can't handle modules
 
-  # Required for el6 clients
 - include_tasks: qpid.yml
-  when: ansible_distribution_major_version == "6"
+  when: ansible_distribution_major_version in ["6", "7"]
```

Comment:
That copr is only published for EL7 (which means this code path was broken before)

Reply:
heh, indeed.

Reply:
updated